risk-map/yaml/risks.yaml

```
@@ -279,6 +354,23 @@ risks:
         - >
           Mitigate this risk by hardening both storage and serving systems to prevent
           unauthorized access and protect against model theft.
+    mappings:
+      mitre-atlas:
+        - AML.T0024.002
+        - AML.T0025
+        - AML.T0048.004
+      stride:
+        - information-disclosure
+    lifecycleStage:
+      - deployment
+      - runtime
+    impactType:
+      - confidentiality
+      - integrity
```

Potential for privacy impact if the model includes privacy data... 

```suggestion
    impactType:
      - confidentiality
      - integrity
      - privacy
```